src/app/modules/group/http-services/create-group-invitations.service.ts

```diff
@@ -16,15 +22,24 @@ export class CreateGroupInvitationsService {
   createInvitations(
     groupId: string,
     logins : string[]
-  ) : Observable<Map<string, any>>
+  ) : Observable<Map<string, InvitationResult>>
   {
     return this.http
       .post<ActionResponse<Object>>(
         `${environment.apiUrl}/groups/${groupId}/invitations`,
         {logins: logins}, {})
       .pipe(
         map(successData),
-        map(objectToMap)
+        map(function (data: Object): Map<string, InvitationResult> {
+          return new Map<string, InvitationResult>(
+            Object.entries(data).map(
+              ([key, value]) => {
+                const result = InvitationResult[value as keyof typeof InvitationResult];
+                if (result == undefined)
+                  throw new Error(`Invitation of user ${key} returned an unexpected result`);
```

Comment:
in your error, you should print the value as well.

Reply:
I used `JSON.stringify(value)` because the value is of type `any`

Reply:
Great 👍